### DISCUSSION TITLE
Antibiotics does not give Drunk Vulnerability

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#13910] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

.

### Reproduction steps

.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for your report, a fix will be included in the next update.

**Reply by Regalis11** [reply]
Could you link to the the issue or commit so we know when to close the discussion (when the fix has been merged)?

**Comment by JeroonBosch** [comment]
[c5a36b0](https://github.com/FakeFishGames/Barotrauma-development/pull/5296/commits/c5a36b0ff8e4be64c3d7cd01fa207ba0c3b843d0) has been merged, closing
